src/NServiceBus.Transport.Msmq.AcceptanceTests/When_setting_label_generator.cs

```diff
@@ -76,7 +76,8 @@ public Endpoint()
                 EndpointSetup<DefaultServer>(c =>
                 {
                     c.AuditProcessedMessagesTo("labelAuditQueue");
-                    c.UseTransport<MsmqTransport>().ApplyLabelToMessages(GetMessageLabel);
+                    var transportSettings = (MsmqTransport)c.ConfigureTransport();
+                    transportSettings.ApplyLabel = GetMessageLabel;
```

Comment:
Could be more explicit as `ApplyCustomLabelToOutgoingMessages`

Reply:
yeah 👍

Reply:
changed